bindings/capacitorjs/src/definitions.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import type { EventType, AccountManagerOptions } from './types';
+export interface IotaWalletMobileTypes {
+    initLogger(path: string): Promise<void>;
+    messageHandlerNew(messageOptions: AccountManagerOptions): Promise<{
+        messageHandler: number;
+    }>;
+    sendMessage(messageOptions: {
+        message: string;
+        handler: number;
+    }): Promise<{
+        result: string;
+    }>;
+    listen(options: {
+        eventTypes: EventType[];
+        messageHandler: number;
+    }, callback: (message: {
```

Comment:
Based on the code in AccountManager.ts, I would expect the following:
```suggestion
    listen(
        eventTypes: EventType[],
        callback: (message: {
```

Reply:
Same as before, please test with your way, it will not work, it's the Capacitor plugin API way.